postgresql.go

```diff
@@ -217,6 +221,9 @@ func (h *PostgreSQL) makeConstraintsDeferrable(db *sql.DB, loadFn loadFunction)
 func (h *PostgreSQL) disableReferentialIntegrity(db *sql.DB, loadFn loadFunction) (err error) {
 	// ensure sequences being reset after load
 	defer func() {
+		if h.DontResetSequences {
```

Comment:
We should actually avoid deferring this whole function if sequences resetting is being skipped:

```go
if h.SkipResetSequences {
	defer func() {
		// ...
	}()
}
```

Reply:
see my previous comment. I think its better to keep the behaviour backward compatible.

Reply:
@dgsb  Sorry, there was a typo in the snippet above. It should be:

```go
if !h.SkipResetSequences {
```

Instead of:

```go
if h.SkipResetSequences {
```

The value is still negated (and backwards compatible). I just suggested renaming it from `Dont` to `Skip`

Reply:
ok, I read to fast your answer